pyproject.toml

```diff
@@ -7,7 +7,17 @@ name = "truststore"
 authors = [{name = "Seth Michael Larson", email = "sethmichaellarson@gmail.com"}]
 readme = "README.md"
 license = {file = "LICENSE"}
-classifiers = ["License :: OSI Approved :: MIT License"]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: MacOS",
+  "Operating System :: Microsoft",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
```

Comment:
Let's wait to claim support for 3.11 until we can successfully test it.

```suggestion
```

Reply:
Done!